docker/build-gcmd-sitemap.sh

```diff
@@ -0,0 +1,5 @@
+
+python index_gcmd.py
```

Comment:
It looks like you have the wrong filename for the script here - please double check that it all works as you would expect it to.

Reply:
oh, I see, that's weird, it runs fine even with the mistake, I would change the file name. Thanks for spotting it.